docker-compose-s3.yml-dist

```diff
@@ -0,0 +1,18 @@
+db:
+  image: postgres:9.3
+client_build:
+  extends:
+    file: docker-compose-base.yml
```

Comment:
Aha, so this 'extends' thing gives us a really simple way to toggle between prod and dev, yes? We just modify the env vars?